src/bridge.js

```diff
@@ -14,19 +14,25 @@ type Message = {|
 export default class Bridge extends EventEmitter {
   _messageQueue: Array<any> = [];
   _timeoutID: TimeoutID | null = null;
+  _destroyed: boolean = false;
   _wall: Wall;
+  _wallUnlisten: Function | null = null;
```

Comment:
I think you added a `null` type here in case a Wall doesn't return an unsubscribe function? But in that case, wouldn't the value actually be `undefined`?

It also looks like maybe all of the listen functions return unlisten functions after this PR? (And you updated the `Wall` type to require this too.)

Reply:
Oops this looks like a leftover from the version where the absence/presense of a wall or unsubscribe function described that the bridge is destroyed/shutdown or not. Now there's a separate flag for that.